src/controllers/favorites.controllers.js

```diff
@@ -28,3 +28,23 @@ export const findAllFavorites = async () => {
     return { status: 'invalid', message: err.message };
   }
 };
+export const deleteFavorite = async (data) => {
+  const { userId, houseId } = data;
+
+  const user = await userService.getById(userId);
```

Comment:
I think that there should be findById.

Reply:
https://szymonkrajewski.pl/the-practical-difference-between-findby-and-getby-in-repositories/

You may use this method in a workflow if you’re sure that the specific entity exists. userId and houseid are required in schema so they always exist. 
![obraz](https://user-images.githubusercontent.com/80857044/155357097-cecb2cde-03cf-446c-9894-859492826c30.png)